cmake/O2Dependencies.cmake

```diff
@@ -27,6 +27,7 @@ find_package(ZeroMQ)
 find_package(AliRoot)
 find_package(FairRoot REQUIRED)
 find_package(FairMQ REQUIRED)
+find_package(Protobuf REQUIRED)
 
```

Comment:
Is it necessary to make protobuf a required dependency? Or can the compilation be conditional? Lets consider protobuf as one way of serialization and not bind the code completely to it.